### PR TITLE
[更新完毕] install

### PR DESCRIPTION
**Pull request by ArkhamWJZ**

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/install.po [hunk 1]
```diff
@@ -45,60 +45,60 @@ msgstr "对于Arch Linux，请执行 ``yaourt -S ncurses5-compat-libs`` 安装
...
 #: ../../install.rst:29
-#, fuzzy
 msgid "Python issues"
-msgstr "OpenGL相关问题"
+msgstr "Python 相关问题"
 
 #: ../../install.rst:31
 msgid "If ``pip`` complains that it could not find a satisfying package, i.e.,"
-msgstr ""
+msgstr "如果 ``pip`` 向你抱怨找不到合适的包，即,"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/install.po]:
```suggestion
msgstr "如果 ``pip`` 报错找不到合适的包，即,"
```

File: locales/zh_CN/LC_MESSAGES/install.po [hunk 2]
```diff
@@ -45,60 +45,60 @@ msgstr "对于Arch Linux，请执行 ``yaourt -S ncurses5-compat-libs`` 安装
...
 
 #: ../../install.rst:33
 msgid ""
-"ERROR: Could not find a version that satisfies the requirement taichi "
-"(from versions: none)\n"
+"ERROR: Could not find a version that satisfies the requirement taichi (from "
+"versions: none)\n"
 "ERROR: No matching distribution found for taichi"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/install.po:84]: Not translated?

File: locales/zh_CN/LC_MESSAGES/install.po [hunk 3]
```diff
@@ -108,68 +108,67 @@ msgstr ""
...
-"of-the-gpu-programming-puzzle/>`_."
+"limited support for `Unified Memory <https://www.nextplatform.com/2019/01/24/"
+"unified-memory-the-final-piece-of-the-gpu-programming-puzzle/>`_."
 msgstr ""
-"这可能是因为你使用的 NVIDIA GPU低于Pascal架构，这对 `统一寻址(Unified Memory) "
-"<https://www.nextplatform.com/2019/01/24/unified-memory-the-final-piece-"
-"of-the-gpu-programming-puzzle/>`_ 会有限制。"
+"这可能是因为你使用的 NVIDIA GPU 低于 Pascal 架构，这对 `统一寻址(Unified "
+"Memory) <https://www.nextplatform.com/2019/01/24/unified-memory-the-final-"
+"piece-of-the-gpu-programming-puzzle/>`_ 会有限制。"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/install.po]:
```suggestion
"这可能是因为你使用的 NVIDIA GPU 低于 Pascal 架构，这对 `统一内存(Unified "
"Memory) <https://www.nextplatform.com/2019/01/24/unified-memory-the-final-"
"piece-of-the-gpu-programming-puzzle/>`_ 会有限制。"
```
I believe **Unified Memory** should be addressed as `统一内存` and `统一寻址` would be more suitable for **Unified Memory Access**? What do you guys think? @archibate @rexwangcc 
Ref: 
1. https://devblogs.nvidia.com/unified-memory-cuda-beginners/#:~:text=What%20is%20Unified%20Memory%3F,on%20either%20CPUs%20or%20GPUs.
2. https://zh.wikipedia.org/wiki/%E5%9D%87%E5%8C%80%E8%AE%BF%E5%AD%98%E6%A8%A1%E5%9E%8B

**Review by isdanni — COMMENTED** [review]
Great work! :rocket:

File: locales/zh_CN/LC_MESSAGES/install.po (hunk 2 above)
Comment (ArkhamWJZ) [reply locales/zh_CN/LC_MESSAGES/install.po:84]: The error message of the terminal is in English. I think keeping the original text may help readers relate to their own situation better 🧐 What's your opinion ,boss？

Comment (isdanni) [reply locales/zh_CN/LC_MESSAGES/install.po:84]: Then follow your decision. :+1:

File: locales/zh_CN/LC_MESSAGES/install.po (hunk 3 above)
Comment (ArkhamWJZ) [reply locales/zh_CN/LC_MESSAGES/install.po]: It does make sense 🧐

File: locales/zh_CN/LC_MESSAGES/install.po [hunk 4]
```diff
@@ -108,68 +108,67 @@ msgstr ""
...
 msgid ""
-"**Possible solution**: add ``export TI_USE_UNIFIED_MEMORY=0`` to your "
-"``~/.bashrc``. This disables unified memory usage in CUDA backend."
+"**Possible solution**: add ``export TI_USE_UNIFIED_MEMORY=0`` to your ``~/."
+"bashrc``. This disables unified memory usage in CUDA backend."
 msgstr ""
-"**解决方案**: 尝试添加 ``export TI_USE_UNIFIED_MEMORY=0`` 到 ``~/.bashrc`` "
-"。该操作将禁用CUDA后端使用统一寻址。"
+"**解决方案**: 尝试添加 ``export TI_USE_UNIFIED_MEMORY=0`` 到 ``~/.bashrc`` 。"
+"该操作将禁用CUDA后端使用统一寻址。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/install.po]:
```suggestion
"该操作将禁用CUDA后端使用统一内存。"
```
Since you already used 统一内存 above.

**Review by archibate — COMMENTED** [review]
LGTMig!

**Review by archibate — APPROVED** [review]
Thank you very much for the quick response!

**Pull request merged**